### PR TITLE
EWL-3405 Begin to implement hover state on navigation

### PR DESCRIPTION
**Pull request by aCyborg**
## JIRA Ticket(s):
- [EWL-3405: HOD Callouts | Navigation](https://issues.ama-assn.org/browse/EWL-3405)

## To Test for cursor/mouse interactions:
- [x] Navigate to Templates > Home > Home
- [x] Observe that mouseover on the primary nav items shows the sub menus for each item
- [x]  Observe that clicking a primary nav item takes you to the overview page (test page)

## To Test for touch interactions:
Think it would be ideal to deploy this somehow in order to test on actual test device, or can try using Safari Web Inspector. https://developer.apple.com/library/content/documentation/AppleApplications/Conceptual/Safari_Developer_Guide/GettingStarted/GettingStarted.html

## Additional Notes:
This changes the main navigation to be hover at 740px+

I tried to maintain on click functionality as well. I need to discuss more with Andy about his expectations around when this shifts form hover to on click and how that works.

### CODE REVIEW

**Comment by froboy** [comment]
@deadarm I'm testing on an iPad and this looks super close. The one item from the ticket that isn't covered is:
> Tap anywhere outside of menu closes menu
I'm working on a solution to that based on some of the JS I had in the codepen linked on the ticket... will def want a review on that though.

File: styleguide/source/assets/js/nav-primary.js [hunk 1]
```diff
@@ -115,6 +115,52 @@ jQuery.noConflict();
       }
     });
 
+    // hover functionality @tablet and greater
+    $('.nav-primary_list-item').hover(
+      // Over
+      function () {
+        if ($(window).width() > 740) {
+          $(this).children('.nav-primary_list-item_title').children('.link-primary-nav').blur();
```
Comment (froboy) [inline styleguide/source/assets/js/nav-primary.js]: I'd like to DRY-ify this if possible in the time that we have. The hover/click open/close seems to be duplicating a lot of lines, and that's not great. It seems like we should be able to add functions for `menuOpen(tab)` and `menuClose(tab)` and then use those for the close behavior I mentioned in the other comment.

**Comment by deadarm** [comment]
@froboy tested this locally and the close is working on my phone. Got into a can of worms trying to refactor, so pushed this fix up for now.

File: styleguide/source/assets/js/nav-primary.js [hunk 2]
```diff
@@ -114,53 +113,18 @@ jQuery.noConflict();
         });
       }
     });
+  });
 
-    // hover functionality @tablet and greater
-    $('.nav-primary_list-item').hover(
-      // Over
-      function () {
```
Comment (froboy) [inline styleguide/source/assets/js/nav-primary.js]: @deadarm where'd all of this go?

**Comment by deadarm** [comment]
Tested on mobile and menu closes when tapping anywhere outside of the menu.

**Comment by froboy** [comment]
@deadarm I remembered the other issue we discussed yesterday. It's that the mobile menu forward/back behavior doens't work as expected, not the open/close. To test:
- View on mobile
- Open menu
- Tap Education
- Tap Back
on my phone, Back doesn't have the proper behavior.

**Comment by deadarm** [comment]
Merging.

**Pull request merged**